test/mpi/configure.ac

```diff
@@ -890,6 +890,14 @@ if test "$have_gpu" = "no" ; then
     AC_SUBST([hip_CPPFLAGS])
     AC_SUBST([hip_LDFLAGS])
     AC_SUBST([hip_LIBS])
+
+    if test "$pac_have_hip" = "yes" ; then
+        PAC_CHECK_HEADER_LIB_OPTIONAL([rccl], [rccl.h], [rccl], [ncclCommInitAll])
```

Comment:
Similar to the `configure.ac`, this line should be `PAC_CHECK_HEADER_LIB_OPTIONAL([rccl], [rccl/rccl.h], [rccl], [ncclCommInitAll],, [-D__HIP_PLATFORM_AMD__])`.